src/applications/disability-benefits/526EZ/components/IntroductionPage.jsx

```diff
@@ -17,11 +17,11 @@ import { toggleLoginModal } from '../../../../platform/site-wide/user-nav/action
 import { focusElement } from '../../../../platform/utilities/ui';
 
 import BetaGate from '../containers/BetaGate';
-
 import { VerifiedAlert } from '../helpers';
-
 import FormStartControls from './FormStartControls';
 
+const gaStartEventName = 'disability-526EZ-start';
```

Comment:
Turns out there's already a `disability-526EZ-sip-form-loaded` event. @djmunoz1 Is that enough, or do we need a new one here?